src/components/MyKiva/LatestBlogCarousel.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<section class="tw-mt-10">
+		<h2 class="tw-text-h3 tw-mb-4">
+			Impact in action
+		</h2>
+		<KvCarousel
+			:slide-max-width="'336px'"
+			:multiple-slides-visible="true"
+			class="blog-card-carousel tw-w-full"
+		>
+			<template v-for="(card, idx) in blogCards" #[`slide${idx}`] :key="card.slug || idx">
+				<BlogCard
+					:image="card.image"
+					:category="card.category"
+					:title="card.title"
+					:date="card.date"
+					:slug="card.slug"
+					@track="trackBlogCard({ title: card.title, slug: card.slug, category: card.category })"
+				/>
+			</template>
+		</KvCarousel>
+	</section>
+</template>
+
+<script setup>
+import { KvCarousel } from '@kiva/kv-components';
+import BlogCard from '#src/components/MyKiva/BlogCard';
+import { inject, watch } from 'vue';
+
+const $kvTrackEvent = inject('$kvTrackEvent');
+
+const props = defineProps({
+	blogCards: {
+		type: Array,
+		required: true,
+	},
+});
+
+watch(
+	() => props.blogCards,
+	newVal => {
+		console.log('blogCards changed :', newVal);
+	},
+	{ deep: true }
+);
+
+function trackBlogCard({ title, slug, category }) {
+	// Replace with your actual tracking function
+	if (typeof $kvTrackEvent === 'function') {
+		$kvTrackEvent('blog-carousel', 'click-card', `${category}:${title}`, slug);
```

Comment:
For the first param of ALL events in this PR we should be using an approved "category" value. In this case it looks like `portfolio` is the most appropriate value..

For the second param in ALL events we should be using an approved "action" value. In this case it should just be `click`.

ref: https://github.com/kiva/snowplow/blob/master/conf/snowplow_allowed_structured_event_values.json

Reply:
Yup, portfolio, click should be the first two strings passed to analytics for mykiva click events.